lib/str.d.ts

```diff
@@ -0,0 +1,585 @@
+declare const Str: {
+    /**
+     * Return true if the string is ending with the provided suffix
+     *
+     * @param str String ot search in
+     * @param suffix What to look for
+     */
+    endsWith(str: string, suffix: string): boolean;
+    /**
```

Comment:
**NAB:style:** can we please have a line break between each function declaration and the comment for the next function?

Reply:
I generated these files with `tsc` as described in other comment. I prefer to leave it as is, it's a lot of manual work that isn't worth IMO.